src/internal/components/menu-dropdown/index.tsx

```diff
@@ -20,6 +20,7 @@ export const ButtonTrigger = React.forwardRef(
   (
     {
       testUtilsClass,
+      internalClass,
```

Comment:
We normally call this `className`, when it is applied to the top element. I would also add a comment, explicitly documenting the use case.

Reply:
Done, thanks!